packages/declarations.d.ts

```diff
@@ -10,3 +10,4 @@ declare module 'chokidar'
 declare module 'sao'
 declare module 'inquirer'
 declare module 'get-port'
+declare module 'puppeteer-core'
```

Comment:
npm install --save @types/puppeteer-core, try this .
declare module is the just a workaround.